src/inference/include/openvino/runtime/intel_gna/gna_properties.hpp

```diff
@@ -0,0 +1,213 @@
+
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief A header for advanced hardware related properties for GNA plugin
+ *        To use in set_property() method of plugins
+ *
+ * @file config.hpp
+ */
+#pragma once
+
+#include "openvino/runtime/properties.hpp"
+
+namespace ov {
+namespace intel_gna {
```

Comment:
we still use the GNAPluginNS namespace in the plugin, don't we need to change it too?

Reply:
@mryzhov I think we can keep `GNAPluginNS`  for legacy code. And use a new one for new code. Anyway, it is better to do it as separate PR.